cg/cli/demultiplex/finish.py

```diff
@@ -17,32 +20,43 @@ def finish_group():
     """Finish up after demultiplexing."""
 
 
-@finish_group.command(name="flow-cell")
-@click.argument("flow-cell-directory-name")
+@finish_group.command(name="illumina-run")
+@click.argument("demultiplexed-run-dir-name")
 @DRY_RUN
 @FORCE
 @click.pass_obj
-def finish_flow_cell(context: CGConfig, flow_cell_directory_name: str, force: bool, dry_run: bool):
-    """Command to finish up a flow cell after demultiplexing.
+def post_process_illumina_run(
+    context: CGConfig, demultiplexed_run_dir_name: str, force: bool, dry_run: bool
+):
+    """Command to post-process an illumina run after demultiplexing.
 
     flow-cell-name is full flow cell name, e.g. '230912_A00187_1009_AHK33MDRX3'.
     """
-    demux_post_processing_api = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api.set_dry_run(dry_run=dry_run)
-    demux_post_processing_api.finish_flow_cell(
-        flow_cell_directory_name=flow_cell_directory_name,
+    post_processing_service = IlluminaPostProcessingService(
+        status_db=context.status_db,
+        housekeeper_api=context.housekeeper_api,
+        dry_run=dry_run,
         force=force,
+        demultiplexed_runs_dir=Path(context.run_instruments.illumina.demultiplexed_runs_dir),
+    )
+    post_processing_service.post_process_illumina_flow_cell(
+        sequencing_run_name=demultiplexed_run_dir_name,
     )
 
 
 @finish_group.command(name="all")
 @click.pass_obj
 @DRY_RUN
-def finish_all_cmd(context: CGConfig, dry_run: bool):
-    """Command to finish up all demultiplexed flow cells."""
-
-    demux_post_processing_api = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api.set_dry_run(dry_run=dry_run)
-    is_error_raised: bool = demux_post_processing_api.finish_all_flow_cells()
+def post_process_all_illumina_runs(context: CGConfig, dry_run: bool):
+    """Command to post process all demultiplexed Illumina runs."""
+
+    post_processing_service = IlluminaPostProcessingService(
+        status_db=context.status_db,
+        housekeeper_api=context.housekeeper_api,
+        dry_run=dry_run,
+        force=False,
```

Comment:
this force needs actually the be implemented into the new post processing flow...

Reply:
ohh ok